test-kitchen.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "mixlib-install",  "~> 0.7"
 
   gem.add_development_dependency "pry"
-  gem.add_development_dependency "winrm-transport", "~> 1.0"
+  gem.add_development_dependency "winrm-transport", "~> 1.0.3"
```

Comment:
Can we do `'~> 1.0', '>= 1.0.3'` instead, so we aren't pinning to a minor version?